### PR TITLE
mediatek: update defconfig files

### PR DESCRIPTION
**Pull request by graysky2**
PR refreshes kernel defconfig files for all tree subtargets under mediatek.  Enabled `CONFIG_ALL_KMODS` and ran make kernel_menuconfig against `mt762[2,3,9]`.  Some of the removed symbols are present in `target/linux/generic/config-5.10` while others were added/removed by the make target.

### CODE REVIEW

**Comment by mans0n** [comment]
"mediatek: " prefix, please.

**Comment by graysky2** [comment]
:roll_eyes: sorry about that.  I am refreshing other subtargets under mediatek.  Will push an update with consistent naming of the commits shortly.

**Comment by dangowrt** [comment]
Please repeat this with **all modules selected** as otherwise you are removing symbols which are masked and may surface once another module or option is selected.

**Comment by graysky2** [comment]
@dangowrt - I added `CONFIG_ALL_KMODS=y` to each and did the refresh.  The corresponding diffs were identical to my original that lacked that.

**Review by dangowrt — APPROVED** [review]

**Comment by graysky2** [comment]
When I test built this today, I am finding that the kernel config step is not working; it requires user interaction to select additional snd modules.  That did not happen when I opened the PR.  Not sure how to handle it.  The behavior is similar to that observed in https://github.com/openwrt/openwrt/pull/5047 which was recently reverted due to [buildbot breakage](https://github.com/openwrt/openwrt/commit/83f2f1ad5840eae4bd265ffc46f1982571514123).

I'm thinking it's best to close this PR until that root cause can be identified and fixed.

File: target/linux/mediatek/mt7623/config-5.10 [hunk 1]
```diff
@@ -260,15 +248,14 @@ CONFIG_HARDIRQS_SW_RESEND=y
 CONFIG_HAS_DMA=y
 CONFIG_HAS_IOMEM=y
 CONFIG_HAS_IOPORT_MAP=y
-CONFIG_HAVE_ARM_ARCH_TIMER=y
```
Comment (dangowrt) [inline target/linux/mediatek/mt7623/config-5.10:263]: There has been some edit war about this one in the past, see 8cdc356f8c30b55698d5f57abbd1f3e16d8c653e

**Comment by graysky2** [comment]
This PR is no longer relevant since the targets have switched to 5.15

**Pull request closed without merging**